scrunch/datasets.py

```diff
@@ -2087,7 +2087,8 @@ def create_crunchbox(self, title='', header='', footer='', notes='',
         if weight is DefaultWeight:
             preferences = self.resource.session.get(
                 self.resource.fragments.preferences)
-            weight = preferences.payload.body.weight or None
+            weight_alias = getattr(preferences.payload.body, "weight_alias", None)
+            weight = None if weight_alias is None else self[weight_alias].url
```

Comment:
If you only need the URL, then doing the `__getitem__` will make an HTTP GET request to the variable entity fetching the full shoji entity only to fetch the URL that we had.

This could be done this way:
```
weight_url = self._catalog.by('alias')[weight_alias].entity_url
```

Reply:
But my question then would be, why does Scrunch needs this URL, why can it not continue with the alias. How is this being used, could we send the alias in that next request?

Reply:
Looks like it should be fine to do `self[weight_alias].url` because `__getitem__` already does what you suggest:
```

    def __getitem__(self, item):
        variable = self._catalog.by('alias').get(item)
        if variable is None:  # Not found by alias
            ...
        return Variable(variable, self)
```